src/lib/service/idp/registration/index.ts

```diff
@@ -67,15 +67,33 @@ export class IdpRegistration extends Base {
 
     /**
      * Validates a previous registration. If successful, the Users account status will be set to 'Active'.
+     * If specified name and password - complete a registration that was not triggered by /v1/register, e.g. via an organization invitation.
      * @param email - Email of the User
      * @param verificationCode - Email verification code
-     * @param regionId - Optional region id
+     * @param name - Optional name of the user
+     * @param password - Optional password of the user
+     * @param company - Optional company of the user
+     * @param language - Optional language of the user
+     * @param timezone - Optional timezone of the user
      * @returns Bearer Token and User object
      */
-    async validateRegistration(email: string, verificationCode: string): Promise<SuccessfulAuth> {
+    async validateOrCompleteRegistration(
+        email: string,
+        verificationCode: string,
+        name?: string,
+        password?: string,
+        company?: string,
+        language?: string,
+        timezone?: string
+    ): Promise<SuccessfulAuth> {
         const resp = await this.axios.patch<User>(this.getEndpoint("/v1/register/verify"), {
             email,
             verificationCode,
+            name,
+            password,
+            company,
+            language,
+            timezone,
```

Comment:
Here instead of this change, please add an optional "validationCode" parameter to the `register` method. This value should be sent in the query parameters. You can use the third argument of the `post` method to send it like this `.post(url, body, { params: { validationCode }})`

Reply:
Thank you Jorge. I have updated this method.